src/gui/tray/unifiedsearchresultslistmodel.h

```diff
@@ -119,6 +122,7 @@ private slots:
 
     QString _searchTerm;
     QString _errorString;
+    bool _waitingForSearchTermEditEnd;
```

Comment:
you are missing an initial value for this bool

Reply:
Fixed in constructor

Reply:
please put it there
that is really easier to read

Reply:
I am not sure what you mean, would you prefer it be placed in the class declaration or in the constructor?